src/include/distributed/connection_management.h

```diff
@@ -30,6 +30,7 @@
 
 /* application name used for internal connections in Citus */
 #define CITUS_APPLICATION_NAME "citus"
+#define CITUS_APPLICATION_NAME_PREFIX "citus-"
```

Comment:
do we still need `CITUS_APPLICATION_NAME`?

Reply:
It is still the fallback application name.
But I'm not sure when it falls back

Reply:
> fallback_application_name
Specifies a fallback value for the application_name configuration parameter. This value will be used if no value has been given for application_name via a connection parameter or the PGAPPNAME environment variable. Specifying a fallback name is useful in generic utility programs that wish to set a default application name but allow it to be overridden by the user.

So, given that we are always going to have an application name, I think it'd be nice to drop `CITUS_APPLICATION_NAME` and all references to it for simplicity.